numpy/polynomial/chebyshev.py

```diff
@@ -856,16 +858,8 @@ def chebpow(c, pow, maxpower=16):
         raise ValueError("Power is too large")
     elif power == 0:
         return np.array([1], dtype=c.dtype)
-    elif power == 1:
-        return c
     else:
-        # This can be made more efficient by using powers of two
-        # in the usual way.
-        zs = _cseries_to_zseries(c)
-        prd = zs
-        for i in range(2, power + 1):
-            prd = np.convolve(prd, zs)
-        return _zseries_to_cseries(prd)
+        return reduce(chebmul, [c]*power)
```

Comment:
I would expect this to be slower, because this expands to something  messy like:
```
        prd = c	
        for i in range(2, power + 1):	
            prd = _cseries_to_zseries(prd)
            zs = _cseries_to_zseries(c)
            prd = np.convolve(prd, zs)
            prd = _zseries_to_cseries(prd)	
```
which performs far more (possibly lossy) conversions back and forth between c and z series

Reply:
I'll test with much larger series, but reduce Performed just as well with small series. I'll post results here in a few hours. If you have any series you think would be good to stress test, feel free to share.

Reply:
Interesting results... https://github.com/jeffyancey/scratchpads/blob/master/polynomial%20pow%20timing.ipynb

Reply:
Did they complete?